Corp/_config.dist.py

```diff
@@ -0,0 +1,8 @@
+# USAGE STRINGS
+SRP_USAGE = 'The commas are important!\nUsage: *!srp [exact pilot name to receive srp], [zkill lossmail url], ' \
```

Comment:
This file should be split out into a private `_config.py` for `SRP_URL` and `SRP_URL_WITH_COMMENT` and a public `config.py` with the data that is save to commit publicly.